### PR TITLE
fixes #97 enforce license checks

### LINKED ISSUE #97: enforce license checks
need to wire up the license check a la app-maven-plugin

### PR DESCRIPTION
**Pull request by jmcc0nn3ll**
Fixes #97

### CODE REVIEW

**Comment by meltsufin** [comment]
It doesn't seem to be applied to `AbstractHelloIntegrationTest.java` and other files in the `test-war-smoke`.

**Comment by jmcc0nn3ll** [comment]
those are in the 10 files committed?

anyway, think you need to add -Ptests if you are building from top, they are still disconnected from mainline build

**Comment by meltsufin** [comment]
Trying this command from `/tests`
```
mvn clean install -Plocal -Djetty.test.image=jetty:9.4
```

**Comment by jmcc0nn3ll** [comment]
and the license check doesn't happen? should pull the check from the parent pom, was complaining for me when I was getting them updated

File: pom.xml [hunk 1]
```diff
@@ -243,6 +243,16 @@
...
             </configuration>
           </execution>
+          <execution>
+            <id>header-check</id>
+            <goals>
+              <goal>check</goal>
+            </goals>
+            <configuration>
+              <configLocation>checkstyle/checkstyle-header.xml</configLocation>
+              <headerLocation>checkstyle/header-java.txt</headerLocation>
```
Comment (meltsufin) [inline pom.xml]: Figured out why it wasn't applied to `*Test.java`. Need to add this line:
```java
              <includeTestSourceDirectory>true</includeTestSourceDirectory>
```

**Pull request merged**